### ISSUE TITLE
A11y_SQLAzureDataStudio_AddAzureAccount_AddAzureAccount_Keyboard:The focus order is not logical after the "select subscription" dropdown has been closed.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9176]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#MAS2.4.3;#Keyboard;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS2.4.3

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Connections from left menu.
4. Select Azure, navigate to + button & hit enter.
5. Navigate to Add account button in right side window & hit enter.
6. Tab to the "select subscription" button.

**Actual:**
On navigating to the button "select subscription" press enter an option list opens on escape it is closed and next tab key focus moves to the new connection under the welcome tab.

**Expected:**
The focus after closing the options list should be on the button "select subscription".

**User Impact:**
Keyboard users will not be able to navigate to the items under the account after a subscription has been selected.

**Attachment link for Reference :**
https://microsoft-my.sharepoint.com/:p:/g/personal/v-stfe_microsoft_com1/Ebnj2g-jhXJIrCCsu3pZHuEBPBKnoG3CuTCHxYZuijr0nw?e=keUw4p

**Comment by alanrenmsft** [comment]
dupe of https://github.com/microsoft/azuredatastudio/issues/9232

**Comment by Roopesh-T** [comment]
The above issue is not repro in the env 
![image](https://user-images.githubusercontent.com/86601127/217233791-ab360247-1c74-4794-aef5-894c56cef280.png)
so closing the bug.
![9176_Closed](https://user-images.githubusercontent.com/86601127/217233897-5023ece7-07b8-4cc7-bcb2-980aa591d3ee.gif)
**GitHub Tag**:#closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
